src/libp2p_transport_tcp.erl

```diff
@@ -250,6 +250,7 @@ fdset(#tcp_state{socket=Socket}=State) ->
     case erlang:get(fdset_pid) of
         undefined ->
             Pid = erlang:spawn(fun() ->
+                                   erlang:put(fsset_for, Parent),
```

Comment:
this is not used anywhere in this PR?

Reply:
I was using it to identify process leaks.